File: 6. Python/EX03.py
# Dados da primeira pessoa
print("# Dados da primeira pessoa: ")
nom1 = input("# Nome: ")        # Nome da primeira pessoa
ida1 = int(input("# Idade: "))  # Idade da primeira pessoa

# Dados da segunda pessoa
print("\n# Dados da segunda pessoa: ")
nom2 = input("# Nome: ")        # Nome da segunda pessoa
ida2 = int(input("# Idade: "))  # Idade da segunda pessoa

# Cálculo da idade média
media = (ida1 + ida2) / 2

# Exibe a idade média
print(f"\n# A idade media de {nom1} e {nom2} = {media:.1f} anos")

File: 4. Java/EX03.java
package Apostila;

import java.util.Locale;
import java.util.Scanner;

public class EX03 
{
    public static void main(String[] args) 
    {
        // Declara variáveis
        String nom1, nom2;
        int ida1, ida2;
        double med;
        
        // Define localização
        Locale.setDefault(Locale.US);
        
        // Cria Scanner
        try (Scanner ent = new Scanner(System.in)) 
        {
            // Dados da primeira pessoa
            System.out.println("# Dados da primeira pessoa:");
            
            System.out.print("# Nome 1: ");
            nom1 = ent.nextLine();
            
            System.out.print("# Idade 1: ");
            ida1 = ent.nextInt();
            
            // Dados da segunda pessoa
            System.out.println("\n# Dados da segunda pessoa:");
            ent.nextLine();
            
            System.out.print("# Nome 2: ");
            nom2 = ent.nextLine();
            
            System.out.print("# Idade 2: ");
            ida2 = ent.nextInt();
        }
        
        // Calcula a média
        med = (ida1 + ida2) / 2.0;
        
        // Exibe a média
        System.out.println("\n# A idade media de " + nom1 + " e " + nom2 + " = "
                            + String.format("%.2f", med) + " anos");
    }    
}
